src/_internalState.ts

```diff
@@ -0,0 +1,36 @@
+import {mouseButtonNumbers} from './mouseButtonNumbers'
```

Comment:
this is the main reason behind the PR - I'd like to make "state" (like button mask and pressed keys) to be tracked by the library, it's a chore to maintain this on the consumer's side and this would match how Playwright works